### PR TITLE
Dark Mode Fixes

### PR DESCRIPTION
**Pull request by Zeeko3**
Polished dark mode, organized CSS for light and dark themes using theme directory. Refactored dark mode to use custom html attribute "data-theme", keeping most of code in CSS rather than JS.

### CODE REVIEW

**Comment by Zeeko3** [comment]
Header color:
{7CF15154-A15A-45B9-8667-5C4E7C4341A0}

Link color:
{BE201D4E-82EB-4FA3-963D-CB05196C6609}

File: application/modules/front/views/layouts/suite.php [hunk 1]
```diff
@@ -11,101 +11,35 @@
   const moon = document.getElementById("moon_theme_toggle_btn");
 
   const toggleThemeButtons = (theme) => {
-    if (theme === "light") {
+    if (theme == 'light') {
```
Comment (Yugi-2) [inline application/modules/front/views/layouts/suite.php:14]: Why change this?

**Comment by Yugi-2** [comment]
The refactor and color changes look good, thanks for this PR. However it looks like the color switch is not working on the main page after this change
image

**Comment by Yugi-2** [comment]
See this issue with white text on light theme
With this change
image

Without:
image


Light theme should be unchanged

**Review by Yugi-2 — APPROVED** [review]
Looks good, thanks for the revisions and for this PR

**Comment by ahadith** [comment]
Jazakumullahu khayr brothers!

**Pull request merged**